src/metabase/models/collection.clj

```diff
@@ -687,10 +687,10 @@
   and write perms for every Group with write perms for the source Collection."
   [source-collection-or-id dest-collections-or-ids]
   ;; figure out who has permissions for the source Collection...
-  (let [group-ids-with-read-perms  (db/select-field :group_id Permissions
-                                     :object (perms/collection-read-path source-collection-or-id))
-        group-ids-with-write-perms (db/select-field :group_id Permissions
-                                     :object (perms/collection-readwrite-path source-collection-or-id))]
+  (let [group-ids-with-read-perms  (t2/select-fn-set :group_id Permissions
```

Comment:
Is this function missing a `:style/indent` rule? is your autoformatter not respecting it? (I sent patches to clj-kondo and cljfmt in January to feed `:style/indent` metadata to cljfmt's indentation rules; make sure you're using recent releases?)

Reply:
in fact, I did update it. I think this `fn` no indent metadata at all :/.

Btw I didn't use my editor